modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -148,17 +149,21 @@ def build_claim_structure(data:, lighthouse_id:, upstream_id:) # rubocop:disable
             claim_type: data[:claim_status_type],
             contention_list: data[:contentions]&.split(','),
             claim_date: data[:claim_dt].present? ? data[:claim_dt].strftime('%D') : nil,
+            current_phase_back: current_phase_back(data),
             decision_letter_sent: map_yes_no_to_boolean(
               'decision_notification_sent',
               data[:decision_notification_sent]
             ),
             development_letter_sent: map_yes_no_to_boolean('development_letter_sent', data[:development_letter_sent]),
             documents_needed: map_yes_no_to_boolean('attention_needed', data[:attention_needed]),
             end_product_code: data[:end_prdct_type_cd],
+            ever_phase_back: ever_phase_back(data),
             jurisdiction: data[:regional_office_jrsdctn],
+            latest_phase_type: latest_phase_type(data),
```

Comment:
For attributes that will only exist on the response in the `show` endpoint, @acovrig and I have been following a pattern of just appending those attributes [here](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb#L68-#L70).

The way it works (at least in my head) is that this `build_claim_structure()` method should really only focus on building the attributes that are common between the `index` and `show` endpoint responses.  Anything else that needs to be added to the `show` response, should just be appended to the common structure over in the `generate_show_output()` method.

So like in your case, `phase_change_date` is going to exist on both endpoint responses. So yeah, let's add that attribute here in `build_claim_structure()`.  But `current_phase_back`, `ever_phase_back`, and `latest_phase_type` are only going to exist on the `show` response.  So let's add those over in `generate_show_output()`.

That make sense?  You ok with that?